datacatalog/cloud-client/src/test/java/com/example/datacatalog/CreateEntryGroupTests.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.datacatalog;
+
+import static org.junit.Assert.assertThat;
+
+import com.google.cloud.datacatalog.EntryGroupName;
+import com.google.cloud.datacatalog.v1beta1.DataCatalogClient;
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import java.util.UUID;
+import org.hamcrest.CoreMatchers;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** Integration (system) tests for {@link CreateEntryGroup}. */
+@RunWith(JUnit4.class)
+@SuppressWarnings("checkstyle:abbreviationaswordinname")
+public class CreateEntryGroupTests {
+
+  private ByteArrayOutputStream bout;
+
+  private static String ENTRY_GROUP_ID =
+      "fileset_entry_group_" + UUID.randomUUID().toString().substring(0, 8);
+  private static String LOCATION = "us-central1";
+  private static String PROJECT_ID = System.getenv().get("GOOGLE_CLOUD_PROJECT");
+
+  @Before
+  public void setUp() {
+    bout = new ByteArrayOutputStream();
+    System.setOut(new PrintStream(bout));
+  }
+
+  @After
+  public void tearDown() {
+    System.setOut(null);
+    bout.reset();
+
+    try (DataCatalogClient dataCatalogClient = DataCatalogClient.create()) {
+      dataCatalogClient.deleteEntryGroup(
+          EntryGroupName.of(PROJECT_ID, LOCATION, ENTRY_GROUP_ID).toString());
+    } catch (Exception e) {
+      System.out.println("Error in cleaning up test data:\n" + e.toString());
+    }
+  }
+
+  @Test
+  public void testCreateEntryGroup() {
+    CreateEntryGroup.createEntryGroup(PROJECT_ID, ENTRY_GROUP_ID);
```

Comment:
It's okay for all the tests to be in a single file. We prefer the snippets themselves in separate files to make them easier to link from our docs, but it's fine if the tests are combined.

Reply:
Since the createEntry is dependant on createEntryGroup, I kept only the CreateFilesetEntryTests, hence it's using both samples files. Is it ok?